py3status/module.py

```diff
@@ -412,6 +442,14 @@ def process_composite(self, response):
             elif urgent and 'urgent' not in item:
                 item['urgent'] = urgent
 
+            # set background and border colors
+            for key, value in self.py3_module_options['block'].items():
+                if (key == 'border_left' and index != 0) or (
+                        key == 'border_right' and index != length):
```

Comment:
what are we doing here a comment is always nice.

something like `we only want side borders at start/end of a composite`